src/client/renderer/r_types.h

```diff
@@ -398,7 +397,6 @@ typedef struct r_material_s {
 	r_image_t *tintmap;
 
 	uint32_t time;
-	uint32_t flags; // these may differ from cm->flags
```

Comment:
Yea, it was these that were fucking me for like 2 days with the animations not working ;)